counter/src/test/resources/config/invalid.xml

```diff
@@ -7,7 +7,7 @@
     <cache-container default-cache="default">
         <transport/>
         <global-state>
-            <persistent-location path="${java.io.tmpdir}/counters"/>
+            <persistent-location path="${java.io.tmpdir}/counters-invalid"/>
```

Comment:
So I am guessing this is the actual fix? I didn't realize there was _another_ test writing to the same directory :(

Reply:
I believe the actual fix is changing the test so that it doesn't start the cache manager. But I thought I'd also change the XML so that others (or myself) won't look at these 2 XMLs using the same path at some point in the future and think it's ok to do the same.